training_pipeline/apps/gradio/semantic_segmentation/palette.txt

```diff
@@ -1,35 +1,3 @@
-[0, 0, 0]
-[216, 82, 24]
-[255, 255, 0]
-[125, 46, 141]
-[118, 171, 47]
-[161, 19, 46]
-[255, 0, 0]
-[0, 128, 128]
-[190, 190, 0]
-[0, 255, 0]
-[0, 0, 255]
-[170, 0, 255]
-[84, 84, 0]
-[84, 170, 0]
-[84, 255, 0]
-[170, 84, 0]
-[170, 170, 0]
-[170, 255, 0]
-[255, 84, 0]
-[255, 170, 0]
-[255, 255, 0]
-[33, 138, 200]
-[0, 170, 127]
-[0, 255, 127]
-[84, 0, 127]
-[84, 84, 127]
-[84, 170, 127]
-[84, 255, 127]
-[170, 0, 127]
-[170, 84, 127]
-[170, 170, 127]
-[170, 255, 127]
-[255, 0, 127]
-[255, 84, 127]
-[255, 170, 127]
\ No newline at end of file
+[0, 10, 146]
+[38, 0, 44]
+[255, 232, 0]
```

Comment:
Short and simple. 

Maybe add a comment on top of this? 

* 0 for background 
* 1 for dog
* 2 for cat